.github/workflows/test-pr.yml

```diff
@@ -1,21 +1,24 @@
 name: Test PR
 on:
-  merge_group:
-  pull_request:
-    paths:
-      - "crates/**"
-      - "packages/**"
-      - "Cargo.toml"
-      - "Cargo.lock"
-      - "rust-toolchain.toml"
-      - "package.json"
-      - "pnpm-lock.yaml"
-      - "pnpm-workspace.yaml"
+  push:
     branches:
-      - "**"
-      - "!release-**"
-    tags-ignore:
-      - "**"
+      - refactor-release
+  # merge_group:
+  # pull_request:
+    # paths:
+      # - "crates/**"
+      # - "packages/**"
+      # - "Cargo.toml"
+      # - "Cargo.lock"
+      # - "rust-toolchain.toml"
+      # - "package.json"
+      # - "pnpm-lock.yaml"
+      # - "pnpm-workspace.yaml"
+    # branches:
```

Comment:
Disabling these seems not expected?

Reply:
I'll uncomment it once everything else is good. This is used for testing.

Reply:
LGTM